primer3/wrappers.py

```diff
@@ -244,86 +297,63 @@ def assessOligo(seq):
 
 
 # ~~~~~~~ RUDIMENTARY PRIMER3 MAIN WRAPPER (see Primer3 docs for args) ~~~~~~ #
-
-if sys.version_info[0] > 2:
```

Comment:
As an aside (probably a separate PR) -- `primerdesign_helpers.c/h` has affordances for Python 2.x support that we should rip out

Reply:
Updated `primerdesing_helpers.c` to remove Python 2 macros